src/Illuminate/Container/composer.json

```diff
@@ -16,6 +16,7 @@
     "require": {
         "php": "^7.1.3",
         "illuminate/contracts": "5.8.*",
+        "illuminate/support": "5.8.*",
```

Comment:
Hello here.
Sorry for the late feedback. But I just realized this new dependency here. I consider this dependency is too heavy to use just `Arr::wrap` method in the container. I would like to have container as lightweight as possible.
Thank you for your attention.

Reply:
Hi, yes, I agree with that, but if we remove this dependency, then we need duplicate is_array check in https://github.com/illuminate/container/blob/160d51a4cc91c1da401c36d7d6e5b2e6ef9afd07/Container.php#L147
and in
https://github.com/illuminate/container/blob/160d51a4cc91c1da401c36d7d6e5b2e6ef9afd07/ContextualBindingBuilder.php#L66
places. I doubt which way is more incorrect.